tests/test_jwud_service.py

```
@@ -770,3 +854,79 @@ def test_a_bare_split_name_with_nothing_on_disk_keeps_its_stem(tmp_path):
+    existing.write_bytes(b"the user's prior output")
+
+    for attr in ("chd_volumes", "data_mount_root"):
+        monkeypatch.setattr(
+            job_manager_module.source_companions_are_safe.__globals__["settings"],
+            attr,
+            str(volume),
+        )
+
+    called = False
+
+    async def fake_convert(*_args, **_kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `fake_convert`

(ANN202)